2020/12xxx/CVE-2020-12510.json

```diff
@@ -1,18 +1,113 @@
 {
```

Comment:
A language tag was used multiple times: description.

Reply:
Fixed, fyi that multi used tag is default behaviour of vulnogram.